api_app/analyzers_manager/file_analyzers/cape_sandbox.py

```diff
@@ -0,0 +1,69 @@
+import time
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class CAPEsandbox(FileAnalyzer):
+    def set_params(self, params):
+        self.__token = self._secrets["token"]
+        self.__vm_name = params.get("VM_NAME", "win7x64_8")
+
+    def run(self):
+        API_URL: str = "https://www.capesandbox.com/apiv2/tasks/create/file/"
```

Comment:
can we add the chance to customize the URL? in this way we can use this for private intances too

Reply:
^makes sense. @0x0elliot u can define the URL in the `docker/.env_template` and then define a variable in the `secrets`. see cuckoo analyzers for reference.

Reply:
i did it the way cuckoo did it.